client/src/tokens/typography.stories.tsx

```diff
@@ -42,7 +42,7 @@ export const TypeScale = () => (
         <tr key={textStyle}>
           <td>
             <span className={`${textStyle} w-mt-4`}>
-              {textStyle.replace('w-', '')}
+              {typeScale[textStyle].text}
```

Comment:
we could set up a global object like
```javascript
const exampleText = {
'w-label-3': 'Medium label',
// ... all the others
};
```

Then here we can adjust to read from that example and default to the replace if needed.

```suggestion
              {example[textStyle] || textStyle.replace('w-', '')}
```